### PR TITLE
The ability to handle several config files

### PR DESCRIPTION
**Pull request by evgenijolshanskij**

### CODE REVIEW

File: src/plugins/WebpackPlugin.ts [hunk 1]
```diff
@@ -253,7 +253,7 @@ const createConfig = (builder: Builder, spin: Spin) => {
           index.push('webpack/hot/signal.js');
         }
       }
-      index.push(builder.entry || './src/server/index.js');
+      index.push(builder.entry || './src/index.js');
```
Comment (vlasenko) [inline src/plugins/WebpackPlugin.ts]: This is a breaking change, it cannot be done without increasing major spinjs version. Is it worth it?

File: src/configRc.ts [hunk 2]
```diff
@@ -14,6 +15,13 @@ export default class ConfigRc {
   public plugins: object[];
 
   constructor(plugins, argv) {
+    const finder = new FileFinder({
+      srcDir: process.cwd(),
+      exclude: ['node_modules', 'flow-typed', 'build'],
```
Comment (vlasenko) [inline src/configRc.ts]: Why specific dirs like `flow-typed` or `build` are ignored here? They are specific to the type of project. Spinjs is a generic tool

File: src/plugins/WebpackPlugin.ts (hunk 1 above)
Comment (evgenijolshanskij) [reply src/plugins/WebpackPlugin.ts]: I made this change long time ago so it is not relevant anymore. Going to revert this modification.

File: src/configRc.ts (hunk 2 above)
Comment (evgenijolshanskij) [reply src/configRc.ts]: Well... So far I've added them for testing purposes. I totally realize that project structure could be completely different.

Comment (vlasenko) [reply src/configRc.ts]: ok

**Pull request closed without merging**